code/game/objects/items/weapons/storage/wallets.dm

```diff
@@ -69,6 +70,8 @@
 	else
 		return ..()
 
+
```

Comment:
??

Reply:
I dunno man. That happens all the time. I type a line one place and further down somewhere gains an empty line. I assumed it was normal.

Reply:
it is not